mycroft/stt/__init__.py

```diff
@@ -22,6 +22,12 @@
 from mycroft.configuration import ConfigurationManager
 from mycroft.util.log import getLogger
 
+
+#imports for json handling, requests and string formatting for KaldiSTT
```

Comment:
no need for comments for imports... btw, you probably will revisit those with the later changes.